_sass/_main.scss

```diff
@@ -115,7 +121,7 @@ a {
     }
   }
 
-  span {
+  b {
```

Comment:
This should not be `b` unless the styling is, literally, `bold`. In this case the styling might change over time, and in any base isn't `bold`. You could argue for `<strong>` instead of `<span>` if you like. `<span>` is less opinionated, but in this case the percentage could be argued as the most important part of those quotes, so could be OK.